### PR TITLE
Added new etc/equipment up to december 2022

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: renewal

* **Description of Pull Request**:

### CODE REVIEW

**Comment by attackjom** [comment]
description of "Good spell / Evil Spell" said  like 
-> When equipped with the Crown of Good and Evil and the Weapon of Good, the ability is displayed.

I think their combo should include weapon too.

File: db/re/item_combos.yml [hunk 1]
```diff
@@ -30385,3 +30385,806 @@ Body:
...
+    Script: |
+      bonus bLongAtkRate,20;
+      bonus2 bSkillAtk,"NW_MAGAZINE_FOR_ONE",15;
+  - Combos:
+      - Combo:
+          - GoodnEvil_Circlet_NW3   # 400393
+          - Injustice_Power   # 312021
+    Script: |
+      bonus bLongAtkRate,20;
+      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT_GUN_GATLING",15;
```
Comment (attackjom) [inline db/re/item_combos.yml:31096]:
```suggestion
      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT",15;
```

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -30385,3 +30385,806 @@ Body:
...
+  - Combos:
+      - Combo:
+          - GoodnEvil_Circlet_NW3   # 400393
+          - RG_5649   # 830025
+    Script: |
+      .@r_weapon = getequiprefinerycnt(EQI_HAND_R);
+      bonus bLongAtkRate,5*(.@r_weapon/3);
+      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT_GUN_GATLING",4*(.@r_weapon/3);
```
Comment (attackjom) [inline db/re/item_combos.yml:31104]:
```suggestion
      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT",4*(.@r_weapon/3);
```

File: db/re/item_combos.yml [hunk 3]
```diff
@@ -30385,3 +30385,806 @@ Body:
...
+      if (getenchantgrade(EQI_HAND_R)>=ENCHANTGRADE_B) {
+         bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT_GUN_GATLING",10;
```
Comment (attackjom) [inline db/re/item_combos.yml:31106]:
```suggestion
         bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT",10;
```

File: db/re/item_combos.yml [hunk 4]
```diff
@@ -30385,3 +30385,806 @@ Body:
...
+      if (getenchantgrade(EQI_HAND_R)>=ENCHANTGRADE_B) {
+         bonus2 bSkillAtk,"NW_MAGAZINE_FOR_ONE",10;
+      }
+  - Combos:
+      - Combo:
+          - GoodnEvil_Circlet_NW4   # 400394
+          - Justice_Power   # 312014
+    Script: |
+      bonus bLongAtkRate,20;
+      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT_GUN_GATLING",15;
```
Comment (attackjom) [inline db/re/item_combos.yml:31125]:
```suggestion
      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT",15;
```

File: db/re/item_combos.yml [hunk 5]
```diff
@@ -30385,3 +30385,806 @@ Body:
...
+         bonus2 bSkillAtk,"NW_WILD_FIRE",10;
+      }
+  - Combos:
+      - Combo:
+          - GoodnEvil_Circlet_NW4   # 400394
+          - Road_Of_Glory   # 820021
+    Script: |
+      .@r_weapon = getequiprefinerycnt(EQI_HAND_R);
+      bonus bLongAtkRate,5*(.@r_weapon/3);
+      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT_GUN_GATLING",4*(.@r_weapon/3);
```
Comment (attackjom) [inline db/re/item_combos.yml:31151]:
```suggestion
      bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT",4*(.@r_weapon/3);
```

File: db/re/item_combos.yml [hunk 6]
```diff
@@ -30385,3 +30385,806 @@ Body:
...
+      if (getenchantgrade(EQI_HAND_R)>=ENCHANTGRADE_B) {
+         bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT_GUN_GATLING",10;
```
Comment (attackjom) [inline db/re/item_combos.yml:31153]:
```suggestion
         bonus2 bSkillAtk,"NW_THE_VIGILANTE_AT_NIGHT",10;
```

**Comment by Atemo** [comment]
Parser updated, thanks!

**Pull request merged**